<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="group_dataset">
     <property name="title">
      <string>数据集</string>
     </property>
     <layout class="QGridLayout" name="gridDataset">
      <item row="0" column="0">
       <widget class="QLabel" name="label_dataset_dir">
        <property name="text">
         <string>保存目录</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="dataset_dir_edit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="choose_dataset_dir_button">
        <property name="text">
         <string>选择…</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_last_img_dir">
        <property name="text">
         <string>上次图片目录</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="last_img_dir_edit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="choose_last_img_dir_button">
        <property name="text">
         <string>选择…</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="auto_save_checkbox">
        <property name="text">
         <string>自动保存</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_roi">
     <property name="title">
      <string>ROI</string>
     </property>
     <layout class="QGridLayout" name="gridRoi">
      <item row="0" column="0">
       <widget class="QCheckBox" name="fix_roi_checkbox">
        <property name="text">
         <string>锁定 ROI 大小</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_roi_w">
        <property name="text">
         <string>宽</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="roi_w_spin">
        <property name="minimum">
         <number>16</number>
        </property>
        <property name="maximum">
         <number>4096</number>
        </property>
        <property name="singleStep">
         <number>16</number>
        </property>
        <property name="value">
         <number>640</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_roi_h">
        <property name="text">
         <string>高</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="roi_h_spin">
        <property name="minimum">
         <number>16</number>
        </property>
        <property name="maximum">
         <number>4096</number>
        </property>
        <property name="singleStep">
         <number>16</number>
        </property>
        <property name="value">
         <number>640</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer_bottom">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Ok|QDialogButtonBox::StandardButton::RestoreDefaults</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl_buttons"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>roi_h_spin</sender>
   <signal>editingFinished()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>setRoiH()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>roi_w_spin</sender>
   <signal>editingFinished()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>setRoiW()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>auto_save_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>setAutoSave(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fix_roi_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>setFixedRoi(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>choose_dataset_dir_button</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>setSaveDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>choose_last_img_dir_button</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>setLastImageDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setRoiW()</slot>
  <slot>setRoiH()</slot>
  <slot>setAutoSave(bool)</slot>
  <slot>setSaveDir()</slot>
  <slot>setFixedRoi(bool)</slot>
  <slot>setLastImageDir()</slot>
 </slots>
</ui>
